v2/definitions/VideoJobResponse.js

```diff
@@ -0,0 +1,216 @@
+module.exports = {
+  type: 'object',
+  description: 'The response to a video request',
+  required: ['id', 'createdAt', 'status', 'video', 'fps', 'jobResults', 'imageResults'],
+  properties: {
+    id: {
+      type: 'string',
+    },
+    createdAt: {
+      type: 'integer',
+      format: 'int64',
+      description: 'UNIX timestamp for when the job was created',
+    },
+    status: {
+      $ref: '#/definitions/JobStatus',
+    },
+    video: {
+      type: 'string',
+      description: 'URL of the video',
+    },
+    fps: {
+      type: 'integer',
+      description: 'Number of frames classified, per second.',
+    },
+    customId: {
+      type: 'string',
+      description:
+        'An arbitrary client specific resource identifier to reference this job (usually UUID)',
+    },
+    feedId: {
+      type: 'string',
+      description: 'User specified ID to reference the source of the images within this job',
+    },
+    webhookUrl: {
+      type: 'string',
+      description: 'Publicly accessible POST endpoint for receiving job status updates',
+    },
+    jobResults: {
+      type: 'array',
+      items: {
+        type: 'object',
+        required: ['class', 'hitl'],
+        properties: {
+          class: {
+            type: 'string',
+            description: 'A class you want to search for',
+          },
+          image: {
+            $ref: '#/definitions/ImageResponseItem',
+          },
+          hitl: {
+            type: 'boolean',
+            description: 'Whether human-in-the-loop verification was used for this class',
+          },
+        },
+      },
+    },
+    imageResults: {
+      type: 'array',
+      items: {
+        type: 'object',
+        required: ['id', 'url', 'status', 'objects'],
+        properties: {
+          id: {
+            type: 'string',
+          },
+          url: {
+            type: 'string',
+            description: 'URL of the image',
+          },
+          proxyUrl: {
+            type: 'string',
+            description: 'Proxy URL of the image',
+          },
+          customId: {
```

Comment:
How does `customId` work here? Since it's only specified for the job, not the video

Reply:
`customId` in the item level is the id of each frame... as per Dom